src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java

```diff
@@ -54,4 +54,18 @@ public static int calculateQuestionnaireType(String treatmentCode) {
   public static boolean isQuestionnaireWelsh(String treatmentCode) {
     return (treatmentCode.startsWith("HH_Q") && treatmentCode.endsWith("W"));
   }
+
+  public static boolean isIndividualQuestionnaireType(String questionnaireId) {
```

Comment:
We've tended to follow a pattern of Maps or Sets for this sort of thing, 
i.e.
  static final Set<String> individualThings =  new HashSet<>(
          Arrays.asList("21", "22", "23", "24"));

  if( individualThings.contains(questionnaireId.substring(0, 2) ) {
        do x;

  See example in fulfilmentRequestService.